parachain-template/runtime/src/lib.rs

```diff
@@ -752,15 +764,15 @@ impl_runtime_apis! {
 		) {
 			use frame_benchmarking::{list_benchmark, Benchmarking, BenchmarkList};
 			use frame_support::traits::StorageInfoTrait;
-			use frame_system_benchmarking::Pallet as SystemBench;
-			use cumulus_pallet_session_benchmarking::Pallet as SessionBench;
+			// use frame_system_benchmarking::Pallet as SystemBench;
```

Comment:
@librelois I was unable to make these benchmarks work with --features runtime-benchmarks
`SystemBench`, `CollatorSelection` complains of unable to guess types
`cumulus_pallet_session_benchmarking` requires `pallet_session::Config` to be implemented which leads to other errors.

Any help is appreciated

Reply:
@nbaztec it's juts a template for example purposes, we could even delete this template entirely because nobody uses it. So doesn't waste time trying to make clean benchmarks for this template,
If it's to compile benchmarks of other crates, disable this template when compiling the benchmarks